components/ResultLoadingUi.tsx

```diff
@@ -0,0 +1,12 @@
+export default function ResultLoadingUi() {
```

Comment:
```suggestion
export default function ResultLoading() {
```

I wouldn't put "UI" in a component name ever since it's redundant